src/frame.rs

```diff
@@ -893,6 +898,50 @@ impl Drop for Frame {
     }
 }
 
+pub struct FrameAtomIterator<'a> {
```

Comment:
More typical rust naming would be `AtomIter` (we are already inside the `frame` module)